demo/src/PaymentPage.jsx

```diff
@@ -23,15 +23,16 @@ class PaymentPage extends React.Component {
     }
   }
 
-  cardNonceResponseReceived = (errors, nonce, cardData) => {
+  cardNonceResponseReceived = (errors, nonce, cardData, token) => {
```

Comment:
Do we want to call this `buyerVerificationToken` or something more descriptive? `token` could be confused with `nonce`, since a `nonce` is technically a type of token.

Reply:
I agree. Let's call it `buyerVerificationToken` to be explicit.